src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,15 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value > 0) {
```

Comment:
try to do this check first and if true then return "0" and if not then we are moving to while. This way your code would be understandable